src/main.rs

```diff
@@ -208,76 +203,56 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         panic!("--immutable can only be used with --cdawg");
     }
 
-    if args.cdawg {
-        return match args.disk_path.clone() {
-            Some(path) => {
-                if args.ram {
-                    println!("Building CDAWG in RAM but saving on disk...");
-                    type Mb = RamBacking<N, (DefaultIx, DefaultIx), DefaultIx>;
-                    let mb = Mb::default();
-                    return Ok(build_cdawg::<Mb>(args, mb)?);
-                }
-                println!("Building CDAWG on disk...");
-                type Mb = DiskBacking<N, (DefaultIx, DefaultIx), DefaultIx>;
-                let mb = Mb::new(path);
-                Ok(build_cdawg::<Mb>(args, mb)?)
-            }
-            None => {
-                println!("Building CDAWG in RAM...");
-                type Mb = RamBacking<N, (DefaultIx, DefaultIx), DefaultIx>;
-                let mb = Mb::default();
-                Ok(build_cdawg::<Mb>(args, mb)?)
-            }
-        };
+    if args.save_path.is_empty() {
```

Comment:
I think it should be possible to make this a required argument here?
https://github.com/viking-sudo-rm/rusty-dawg/blob/b5ce13a8ec15a259f6f972e6d93de76c28929291/src/main.rs#L78